NooBaa operator pod is in panic sometimes :)

### Environment info
- NooBaa Operator Version: master
- Platform: minikube 

### Actual behavior
1. NooBaa operator pod crashes with a panic error - invalid memory address or nil pointer dereference

2. It won't get reproduced on every installation 
3. This symptom can be solved by returning an error when the connection is not defined but the purpose of opening this issue is to investigate why the RPC connection is not defined.

### Expected behavior
1. No crash

### Steps to reproduce
1. Install NooBaa and hope for NooBaa operator pod to crash with SIGSEGV panic attack

### More information - Screenshots / Logs / Other output
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x46b356]

goroutine 23169 [running]:
fmt.(*buffer).writeString(...)
	/usr/local/go/src/fmt/print.go:82
fmt.(*fmt).padString(0x199?, {0x0, 0x37})
	/usr/local/go/src/fmt/format.go:110 +0x247
fmt.(*fmt).fmtS(0xc0009ad430?, {0x0?, 0x1f5e300?})
	/usr/local/go/src/fmt/format.go:359 +0x3f
fmt.(*pp).fmtString(0xc00199e0d0?, {0x0?, 0x40d71f?}, 0x0?)
	/usr/local/go/src/fmt/print.go:474 +0x86
fmt.(*pp).printValue(0xc00199e0d0, {0x1f5e300?, 0xc0012564a0?, 0x8?}, 0x76, 0x3)
	/usr/local/go/src/fmt/print.go:788 +0x107e
fmt.(*pp).printValue(0xc00199e0d0, {0x2047a60?, 0xc000efd8f0?, 0x0?}, 0x76, 0x2)
	/usr/local/go/src/fmt/print.go:809 +0x1ee5
fmt.(*pp).printValue(0xc00199e0d0, {0x22abf60?, 0xc000efd8c0?, 0x5?}, 0x76, 0x1)
	/usr/local/go/src/fmt/print.go:837 +0x12b4
fmt.(*pp).printValue(0xc00199e0d0, {0x22a57a0?, 0xc000efd8c0?, 0x8?}, 0x76, 0x0)
	/usr/local/go/src/fmt/print.go:907 +0xb85
fmt.(*pp).printArg(0xc00199e0d0, {0x22a57a0?, 0xc000efd8c0}, 0x76)
	/usr/local/go/src/fmt/print.go:743 +0x756
fmt.(*pp).doPrintf(0xc00199e0d0, {0x23dbf6e, 0x12}, {0xc0009adee8?, 0x2, 0x2})
	/usr/local/go/src/fmt/print.go:1057 +0x288
fmt.Sprintf({0x23dbf6e, 0x12}, {0xc000bc1ee8, 0x2, 0x2})
	/usr/local/go/src/fmt/print.go:219 +0x59
github.com/sirupsen/logrus.(*Entry).Logf(0xc00027a3f0, 0x4, {0x23dbf6e?, 0xc00005a218?}, {0xc000bc1ee8?, 0xdf8475800?, 0x3b89820?})
	noobaa-operator/vendor/github.com/sirupsen/logrus/entry.go:349 +0x49
github.com/sirupsen/logrus.(*Logger).Logf(0xc0000e6200, 0x4, {0x23dbf6e, 0x12}, {0xc000bc1ee8, 0x2, 0x2})
	noobaa-operator/vendor/github.com/sirupsen/logrus/logger.go:154 +0x85
github.com/sirupsen/logrus.(*Logger).Infof(...)
	noobaa-operator/vendor/github.com/sirupsen/logrus/logger.go:168
github.com/sirupsen/logrus.Infof(...)
	noobaa-operator/vendor/github.com/sirupsen/logrus/exported.go:199
github.com/noobaa/noobaa-operator/v5/pkg/nb.(*RPCConnWS).ping(0xc000efd8c0?)
	noobaa-operator/pkg/nb/rpc_ws.go:137 +0xd9
github.com/noobaa/noobaa-operator/v5/pkg/nb.(*RPCConnWS).SendPings(0x0?, {0x29678d8, 0xc0011f8f40})
	noobaa-operator/pkg/nb/rpc_ws.go:153 +0x106
created by github.com/noobaa/noobaa-operator/v5/pkg/nb.(*RPCConnWS).ConnectUnderLock
	noobaa-operator/pkg/nb/rpc_ws.go:128 +0x50a
```

--- a/pkg/nb/rpc_ws.go
+++ b/pkg/nb/rpc_ws.go
@@ -134,7 +134,6 @@ func (c *RPCConnWS) ConnectUnderLock() error {
 func (c *RPCConnWS) ping() error {
 	ctx, cancel := context.WithTimeout(context.Background(), pongTimeout)
 	defer cancel()
-	logrus.Infof("RPC: Ping (%p) %+v", c, c)
 	return c.WS.Ping(ctx)
 }
 

